packages/components/src/components/input-text/shadow.tsx

```diff
@@ -20,16 +24,14 @@ import type {
 	SyncValueBySelectorPropType,
 	TooltipAlignPropType,
 } from '../../schema';
-import { buildBadgeTextString, setState, showExpertSlot, validateAlert } from '../../schema';
-import type { JSX } from '@stencil/core';
-import { Component, Element, Fragment, h, Host, Method, Prop, State, Watch } from '@stencil/core';
+import { setState, validateAlert } from '../../schema';
 
 import { nonce } from '../../utils/dev.utils';
 import { propagateSubmitEventToForm } from '../form/controller';
-import { getRenderStates } from '../input/controller';
-import { InternalUnderlinedBadgeText } from '../../functional-components';
+import KolFormFieldFc, { type FormFieldStateWrapperProps } from '../../functional-component-wrappers/FormFieldStateWrapper';
```

Comment:
Grundsätzlich: Ich fände es besser, immer den tatsächlichen Namen für die Import zu verwenden, auch wenn er etwas länger ist.

```suggestion
import FormFieldStateWrapper, { type FormFieldStateWrapperProps } from '../../functional-component-wrappers/FormFieldStateWrapper';
```